src/main/frontend/pipeline-console-view/pipeline-console/main/ConsoleLogStream.tsx

```diff
@@ -20,6 +21,7 @@ export default function ConsoleLogStream(props: ConsoleLogStreamProps) {
   const [moveToBottom, setMoveToBottom] = useState(true);
   const showButtonInterval = useRef<NodeJS.Timeout | null>(null);
   const [showButton, setShowButton] = useState(false);
+  const [consoleLineHeight, setConsoleLineHeight] = useState(1);
```

Comment:
Setting `1` as a default value feels like a bit of a hack, but the component wouldn't render when set to `0` and then the `consoleLineHeightCallback` never gets called.